src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -13,16 +13,15 @@
 
 use Symfony\Component\Debug\Exception\FlattenException;
 use Symfony\Component\Debug\Exception\OutOfMemoryException;
+use Symfony\Component\Debug\Formatter\FormatterInterface;
+use Symfony\Component\Debug\Formatter\HtmlFormatter;
 
 /**
- * ExceptionHandler converts an exception to a Response object.
+ * ExceptionHandler catches exceptions and generates debug output.
```

Comment:
It does not catch exceptions, php does. What about 
`ExceptionHandler pretty prints exceptions to ease debugging.`